### DISCUSSION TITLE
Adding a white(?) border to each emoji to improve legibility

### DISCUSSION THREAD

**Discussion by C1710** [C1710/blobmoji#101] [Style]
There are emojis which are very hard to read on dark (or light) backgrounds.  
Therefore it might make sense to add a border, similar to [how Microsoft did it with their Windows 10 version of Segoe UI Emoji](https://emojipedia.org/microsoft/windows-10-may-2019-update/white-heart/).  
I personally would prefer a white border though, because I think it might look nicer. Maybe a dark/light version could be possible, but I'm not sure.

Right now it's just a thought, I will probably not have time in the next few weeks/months to do this anyway.  
It be completely optional, as a separate version, maybe even only available as an optional feature in `emoji_builder`.

**Comment by Secret-chest** [comment]
I don't support, it will look ugly.
